app/config/deletion_counter.yml

```diff
@@ -0,0 +1,8 @@
+parameters:
+  deletion_counter:
+    enwiki:
```

Comment:
I think the config should maybe go by domain to be consistent with semi_automated.yml and rfx_analysis.yml